modules/manage/partials/schema-compatibility.adoc

```diff
@@ -42,4 +42,24 @@ Remove fields
 | Remove required fields 
 
 Change data types of fields
+
+| **JSON**
+a| Add optional properties
```

Comment:
stray `a`?

Reply:
@andijcr It's an AsciiDoc style operator that is added to the table cell so that the bulleted list is formatted properly.

Reply:
oh right, I'm unfamiliar with the syntax. thanks for the clarification